gems/smithy-client/lib/smithy-client/identities/token.rb

```diff
@@ -3,15 +3,21 @@
 module Smithy
   module Client
     module Identities
-      # Identity class for HTTP Bearer token authentication.
-      class HttpBearer < Identity
+      # Identity class for token authentication.
+      class Token < Identity
         def initialize(token:, **)
           @token = token
           super(**)
         end
 
         # @return [String, nil]
         attr_reader :token
+
+        # Removing the token from the default inspect string.
+        # @api private
+        def inspect
+          "#<#{self.class.name} token=[FILTERED]>"
```

Comment:
I think this would need to include expiration too right? I think a better implementation may just be to gsub out `@token="contents"` with a regex and replace it with filtered.